app/views/users/edit.html.erb

```diff
@@ -1,171 +1,177 @@
-<% content_for :title do %>
-  <%= @is_own_settings ? "My Settings" : "Settings | #{@user.username}" %>
-<% end %>
+<main class="container">
+  <% content_for :title do %>
```

Comment:
I like the new divs here but let's keep the `content_for :title` outside of any nesting like it was before the change to make it clear this is just for the header.

Reply:
Done!